### PR TITLE
[MOB-3176] Cancellation SLA to booking

### PR DESCRIPTION
**Pull request by EdwardW2**
## JIRA
[MOB-3176](https://karhoo.atlassian.net/browse/MOB-3176)

## DESCRIPTION
[Create a description of what you have changed in this branch]

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [x] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [x] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [x] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

**Comment by jsm20** [comment]
Do we not need any changes to the tests for this? I think you might need to update the TripInfo.json and do an assert in the integration tests.

File: KarhooSDK.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (mohadian) [inline KarhooSDK.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:1]: Do we need to have this file as part of this PR?

Comment (EdwardW2) [reply KarhooSDK.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:1]: It was automatically added, just like cocoapods and its pod lock file worked whenever there was a pod install/update ran. Could just gitignore the file though

**Comment by EdwardW2** [comment]
> Do we not need any changes to the tests for this? I think you might need to update the TripInfo.json and do an assert in the integration tests.

JSON updated. Didn't see any tests that needed updating in here nor on Android.

**Review by mohadian — APPROVED** [review]

**Pull request merged**